static/app/views/explore/hooks/useSaveQuery.tsx

```diff
@@ -39,19 +40,19 @@ export function useSaveQuery() {
       environment: environments,
       query: [
         {
+          aggregateField: aggregateFields.map(aggregateField => {
+            return isVisualize(aggregateField) ? aggregateField.toJSON() : aggregateField;
+          }),
```

Comment:
Should we also incorporate the blank string filter for group by as well? If I add a blank group by, it raises an error here and I have to back out to save it